server/pages/index.js

```diff
@@ -319,6 +319,13 @@ module.exports = function() {
 
 	app.get( '/theme', ( req, res ) => res.redirect( '/design' ) );
 
+	app.get( '/plans', function( req, res, next ) {
+		if ( config( 'env' ) !== 'development' && ! req.cookies.wordpress_logged_in ) {
```

Comment:
Looking at a similar code above, why not put the `if` condition above the `app.get` call? That way, we would add a new route + handler only if the condition is truthy, not always as it's now. It's a micro optimization but I'm just wondering whether this was intentional and if yes, why. :)